src/components/cards/ArnsCard/styles.css

```diff
@@ -0,0 +1,38 @@
+.arnsCard {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 24px;
+  row-gap: 30px;
+  width: 198px;
```

Comment:
we should be able to leverage `flex` with `flex-wrap` to properly show three cards per row instead of hard-coding widths

Reply:
not sure if gap can be dynamic but something glike
```suggestion
 width: 30%
 row-gap: 3%
```

Reply:
how does this perform on mobile now?

Reply:
personally I think mobile looks pretty good 
![image](https://user-images.githubusercontent.com/85306700/201711907-8cf176a8-20cc-4967-9958-fe568fab1135.png)